### PR TITLE
Community page redesign

### PR DESCRIPTION
**Pull request by garbas**
## Design

- [Community_deskop.pdf](https://github.com/NixOS/nixos-artwork/blob/master/www/2020_redesign/Community/Community_deskop.pdf)
- [Community_mobile.pdf](https://github.com/NixOS/nixos-artwork/blob/master/www/2020_redesign/Community/Community_mobile.pdf)

## TODO

- [x] use blue buttons for "Discourse forum" and "Open webchat" links in Communication section (@garbas)
- [x] for reddit, discord, twitter use icons and align them in a grid (@garbas)
- [x] _diagonal_ light blue backround (@samueldr)
- [x] add short description for past nixcon (@garbas)
- [x] apply styles for nixcon and adjust to light blue background (@garbas)
- [x] design style for list of meetups (@garbas)

---
This change is [Reviewable](https://reviewable.io/reviews/nixos/nixos-homepage/590)

### CODE REVIEW

**Comment by samueldr** [comment]
Quick notes about the ^^^^ previous push

* * *

The NixCon logo being embedded that way would pretty much double the general CSS file output size. That's not good.

I added what was already planified: the ability to make page-specific styles. The capability here should be used only for things that increase the main styles by too much. Generally speaking, that will be page-specific assets.

* * *

Coupled with that, a white backdrop for the NixCon logo. Since this uses the Nix lambdaflake, and it shouldn't be used against a dark blue backdrop in its blue form, it has been put into a white box. The box emulates the concept presented in the mockup better:

image


Sure, a *photo* type image would have helped, but it's not something we really have here.

**Comment by samueldr** [comment]
The world map asset takes *ages* to run `svgo` against (due to the sheer amount of elements).

This is why the assets optimization has been split-off in the last batch of changes.

File: community.tt [hunk 1]
```diff
@@ -4,83 +4,147 @@
...
+    newsletter.</p>
 </section>
 
-<div class="generic-layout">
-
-  <h2>Communication</h2>
-
-  <div class="columnar-layout -three">
+<section class="community-communication">
+  <h1>Where to talk?</h1>
```
[the remark this answers is not in the document]
Comment (garbas) [reply community.tt:21]: Yeah I'm also struggling to find better title, but is better then "communication". I took this one from https://www.rust-lang.org/community.

**Comment by garbas** [comment]
@samueldr :100:

File: community.tt (hunk 1 above)
[the remark this answers is not in the document]
Comment (samueldr) [reply community.tt:21]: I think the main difference here is the question mark.

**Pull request merged**